androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/Extensions.kt

```diff
@@ -43,3 +44,17 @@ fun Context.setStatusBarVisibility(showStatusBar: Boolean) {
         }
     }
 }
+
+fun Context.setNavigationBarVisibility(showNavigationBar: Boolean) {
+    val window = this.findActivity()?.window
+    window?.let {
```

Comment:
nit: I think you can avoid to create a new var and do `this.findActivity()?.window?.let { ...... }`

Reply:
I just followed the pattern of the other extensions